apps/finance/app/src/components/NewTransfer/Withdrawal.js

```diff
@@ -202,6 +203,16 @@ const CombinedInput = styled.div`
   }
 `
 
+const StyledAsterisk = styled.span.attrs({
+  children: '*',
+  title: 'Required',
+})`
+  color: ${theme.accent};
+  float: right;
```

Comment:
I won't be able to sleep anymore :(
Can I convince you to use `justify-content: space-between` (either `flex` or `grid`) on the parent container?

Reply:
Haha actually this was taken from [@aragon/ui's Field](https://github.com/aragon/aragon-ui/blob/master/src/components/Field/Field.js#L14) but we can look at changing that.

Reply:
See https://github.com/aragon/aragon-ui/pull/275 :)